Controller/MTGAController/Controller.py

```
@@ -6043,17 +6043,36 @@ def _click_done() -> None:
+                        done_tpl = os.path.join(self._buttons_dir(), "scry_done.png")
+                        if os.path.exists(done_tpl) and self._click_image_in_scaled_arena_region(
+                            done_tpl, "SCRY_DONE", rel_region=(700, 820, 520, 240),
+                            confidence=0.78, timeout=1.5,
+                        ):
+                            bot_logger.log_info("GROUP_REQ Done click: matched scry_done.png template.")
+                            return
+                        # Fallback fixed coordinate (base 1920x1080), measured from
+                        # real captures: scry's Done button centres at (960, 878),
+                        # surveil's ~47px lower at (960, 925). Pick by context so a
+                        # template miss still lands on the right overlay's button.
+                        base_point = (960, 878) if "scry" in context.lower() else (960, 925)
```

Ruff (0.15.20)

[warning] Function definition does not bind loop variable `context`

(B023)